mycroft/enclosure/gui.py

```diff
@@ -478,7 +478,8 @@ def stop_video(self):
         # TODO detect end of media playback from gui and call this
         if self.is_video_displayed:
             self["playStatus"] = "stop"
-            self.clear()
+            self.skill.bus.emit(Message("mycroft.gui.screen.close",
```

Comment:
Should this be `self.gui.release()` now?

Reply:
Other than that, it's all working so good to merge :tada

Reply:
self.gui.clear() seems to cause an issue where it clears the data but also causes an issue of sending repeated events to mycroft-gui so its better to send the event directly for now. In any case self.gui.release() was never fully implemented. This PR (https://github.com/MycroftAI/skill-mark-2/pull/57) implements proper handling of the "mycroft.gui.screen.close" event for both this PR and self.gui.release() to work correctly and is needed for this PR as well.